app/views/Main.js

```diff
@@ -103,18 +104,19 @@ const Main = () => {
   return (
     <View style={styles.backgroundImage}>
       {page}
-      <SettingsIcon />
+      <SettingsNavButton />
+      <FeatureFlagNavButton navigation={navigation} />
```

Comment:
What does this look like? below, to the left ? provide a screenshot

Reply:
<img width="388" alt="Screen Shot 2020-05-09 at 4 14 12 PM" src="https://user-images.githubusercontent.com/20157849/81485054-210f3c00-9210-11ea-94bd-6c3d9d5f25a6.png">

Reply:
oof. Sorry to push back again on this... This is pretty heavy and could confuse users as we make screenshots of the main page (in dev mode, it's not easy to remove, so would need commenting out of code..)

How do you feel about 

1. long press on the team icon to get to the flags page?

2. long press or 5 presses on the team icon enables flags on this page, and allows to disable?

I'm leaning towards 1.

2. at least allows us to remove this icon for screenshots.

Reply:
@kenpugsley @efalkner @penrods

Reply:
ah yeah i see how this could be a bottleneck in those scenarios. good point. 

on the flip side though, i do feel like it ends up being a bit of an "insider knowledge" thing with either of those approaches. thinking of a comment that Christian made about "that incantation required to toggle exposure mode".

what do you think of something like this, that would allow us to still take normal screenshots?

<img width="389" alt="Screen Shot 2020-05-09 at 6 44 10 PM" src="https://user-images.githubusercontent.com/20157849/81487671-14e1a980-9225-11ea-8418-fea79661c2f8.png">

Reply:
However - if the concern is just having any sort of UI element that navigates users to the page, then I like idea #1.

Reply:
"Insider" "incantation" is actually my preference (like android dev mode, 7 presses on the build number), so that only internal devs and testers find it. Or select people that we inform.

Your option of the last panel in setting is quite good. I like that. 

I'd consider only showing it _after_ the special incantation. But would certainly consider it being there always in dev builds

Reply:
Go with a row in settings for now